test/libsolidity/syntaxTests/parsing/literal_constants_with_ether_subdenominations.sol

```diff
@@ -5,11 +5,13 @@ contract c {
          b = 2 szabo;
          c = 3 finney;
          b = 4 ether;
```

Comment:
There are two `b`s here and one shadows another. The expected warning below suggest that it's intentional. But why? Doesn't seem crucial to this test.